### PR TITLE
Chore/improve install script

### PR DESCRIPTION
**Pull request by MJLHThomassen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,5 +5,64 @@ This repository was forked from the UBerGames rpgxEF repository located here: ht
...
+Then chose the xubuntu-xx.yy-desktop-amd64.iso image (where xx.yy is 16.04 at the time of writing)
+
+* Run your VM
+* Download the install_full.sh file or copy its contents into a new .sh file somewhere on your VM
+* Open a terminal window in the location where you downloaded the install_full.sh file (For Xubuntu, go to the directory using the File Manager, then right click and select "Open Terminal Here").
+* Type: sudo ./install_full.sh
+* When prompted for a "root password",. insert the password you chose for your user account while installing Linux on your VM
+* The script automatcially downloads the RPG-X EF Repository from GitHub and puts it in /home/<yourusername>/Projects/rpgxef (obviously, <yourusername> is the username of your Linux user account)
+* It also automatically downloads all development dependencies needed for building the source code
+* In the terminal, navigate tot he rpgxef repository directory which was just created: cd /home/<yourusername>/Projects/rpgxef
```
Comment (solarisstar) [inline README.md]: tot he -> to the

File: README.md [hunk 2]
```diff
@@ -5,5 +5,64 @@ This repository was forked from the UBerGames rpgxEF repository located here: ht
...
+## For Advanced Users
+* Git Pull this repository onto a Ubuntu flavoured Linux machine or VM
+* Terminal to the repository directory and run: ./install_deps.sh. This will install all development dependencies for running make
+* Run make to build for Linux 64 bits, find the output files in the build directory
+  * Run ./make-i368.sh for building the Linux 32 bits variant
+  * Run ./cross-make-mingw.sh for building the Windows 32 bits variant
+  * Run ./cross-make-mingw64.sh for building the Windows 64 bits variant
+* Install a fresh copy of the RPG-X Standard Edition from TLO: http://www.last-outpost.net/rpgx/ and put the build files in the correct directory, then run executable file to launch RPG-X EF
+
+  * Copy the following files to the base install directory of RPG-X (where the current .exe is located, overwrite if asked):
```
Comment (solarisstar) [inline README.md]: .exe -> binary

File: README.md [hunk 3]
```diff
@@ -5,5 +5,64 @@ This repository was forked from the UBerGames rpgxEF repository located here: ht
...
+  Then press Apply and OK and start the game via the shortcut
+
+## For Advanced Users
+* Git Pull this repository onto a Ubuntu flavoured Linux machine or VM
+* Terminal to the repository directory and run: ./install_deps.sh. This will install all development dependencies for running make
+* Run make to build for Linux 64 bits, find the output files in the build directory
+  * Run ./make-i368.sh for building the Linux 32 bits variant
+  * Run ./cross-make-mingw.sh for building the Windows 32 bits variant
+  * Run ./cross-make-mingw64.sh for building the Windows 64 bits variant
+* Install a fresh copy of the RPG-X Standard Edition from TLO: http://www.last-outpost.net/rpgx/ and put the build files in the correct directory, then run executable file to launch RPG-X EF
```
Comment (solarisstar) [inline README.md]: in the correct directory as described below

File: README.md [hunk 4]
```diff
@@ -5,5 +5,64 @@ This repository was forked from the UBerGames rpgxEF repository located here: ht
...
+
+  * Copy the following files to the base install directory of RPG-X (where the current .exe is located, overwrite if asked): 
+    * build/release-<platform>-<arch>/rpgxEF.<arch> 
+    * build/release-<platform>-<arch>/renderer_opengl1_<arch> 
+
+  * Copy the following files to the RPG-X2 directory located in your RPG-X install directory:
+    * build/release-<platform>-<arch>/rpgxEF/ui<arch>, 
+    * build/release-<platform>-<arch>/rpgxEF/qagame<arch>
+    * build/release-<platform>-<arch>/rpgxEF/cgame<arch>
```
Comment (solarisstar) [inline README.md]: .so for Linux, .dll for windows

File: install_full.sh [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+fi
+
+# Pull the git repository, put it in a Projects folder in the users home directory
+cd ~ || exit -1
+mkdir -p Projects
+git clone https://github.com/solarisstar/rpgxEF.git
+cd ~/rpgxEF || exit -1
+
+# Install all development dependencies
+./install_deps.sh
```
Comment (solarisstar) [inline install_full.sh]: newline

**Comment by solarisstar** [comment]
Some comments but overall looks good.

**Comment by MJLHThomassen** [comment]
Fixed your comments.

Can this be merged?

**Pull request merged**